search_gov_spiders/search_gov_spiders/spiders/test/crawl_tests.py

```diff
@@ -0,0 +1,17 @@
+from unittest.case import TestCase
```

Comment:
It seems like the tests and mocks should be at the root level and not nested deep down in the `spiders` dir.  Do we have examples to consult to see how this is typically handled?  I would assume that the `spiders` dir should remain a somewhat pristine place where we only add our specifically configured spiders per this architecture drawing: https://docs.scrapy.org/en/latest/topics/architecture.html